tests/ci/cdk/cdk/codebuild/github_ci_android_omnibus.yaml

```diff
@@ -47,5 +47,5 @@ batch:
       env:
         type: LINUX_CONTAINER
         privileged-mode: true
-        compute-type: BUILD_GENERAL1_MEDIUM
+        compute-type: BUILD_GENERAL1_LARGE
```

Comment:
I guess this is a leftover of a merge with main?

Reply:
@nebeid actually pushed this change in https://github.com/aws/aws-lc/pull/1273/commits/506dceda886bb739a46a000c9b3f83f3e7999fd9. I can't see the Android CI runs, I think, but when she pushed this it started passing.

Reply:
That's right. An advice from the team was to increase the capacity for that particular Android cross build which wasn't passing.